rokka/client-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "\\Rokka\\RokkaClientBundle\\RokkaClientBundle": ["all"]
```

Comment:
the leading slash is not neeeded and not used in the other recipes

Reply:
@fabpot should we enforce it in the validator ?

Reply:
fixed that one. And yes, would be great, if the validator would catch that